services/sidecar/Dockerfile

```diff
@@ -95,21 +104,17 @@ RUN $SC_PIP install -r requirements/prod.txt &&\
 #  + /home/scu     $HOME = WORKDIR
 #    + services/sidecar [scu:scu]
 #
-FROM cache as production
+FROM base as production
 
 
-ENV SC_BUILD_TARGET production
+ENV SC_BUILD_TARGET=production
 ENV PYTHONOPTIMIZE=TRUE
 
 WORKDIR /home/scu
 
-RUN mkdir -p services/sidecar &&\
-    chown scu:scu services/sidecar &&\
-    mv /build/services/sidecar/docker services/sidecar/docker &&\
-    rm -rf /build
-
-RUN apk del --no-cache\
-      gcc
+COPY --from=cache ${VIRTUAL_ENV} ${VIRTUAL_ENV}
```

Comment:
I guess this is enough because of the new base image, right? In the old times with alpine we had lots of issues with lib dependencies

Reply:
well I started by doing this in alpine and it worked as well.
Used the info from https://pythonspeed.com/articles/multi-stage-docker-python/

Reply:
in alpine it did not always work. One good test is taking libraries with externals like ``ujson, pandas, numpy, ...`` .

Reply:
well I did not detect anything weird yet.